app/pulp/app/serializers/catalog.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import serializers
+
+from django.core import validators
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class DownloadCatalogSerializer(ModelSerializer):
+    url = serializers.CharField(
+        help_text="The URL used to download the related artifact.",
+        allow_blank=True, required=False, validators=[validators.URLValidator],
```

Comment:
So, to make it clear what change is requested here: This field should be read-only, and does not need validation.